back-office/wishlist.md

```diff
@@ -0,0 +1,337 @@
+
+Products with several options:
+- if I delete my product from the product sheet: the product with combinations X and quantity Y will be deleted
+- if I delete my product from a product listing (where I don't have any variation options), it deletes only the product with the defaultvariation. Anyway, if I have added a declination as a favourite, then the product block in the listing will not have the heart filled. The heart is filled only if the merchant has added the product with its default declination. 
+
+Exemple : if I didn't added the product with the default combination in my favorite list
+For example I have a tshirt with following combinations:
+* Red (default combination)
+* Black
+* White
+I add the black and the white to one of my favorite list and tt will be impossible to remove the product from the wishlist because it is not be considered as a favorite product (the heart is empty). 
+
+### US 4: Add a favorite product to the cart  
+##### As a logged-in user, I can add a favorite product to the shopping cart
+
+Specific case:
+-	customized products cannot be added directly to the cart from the list of favorite products
+
+Functional rules:
+A logged-in user can add one of their favorites to the cart, except customized products (with stock or out of stock but with the possibility to be added to the cart/allow order option), by clicking on Add to cart.
+
+![wishlist screen 11](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-11.png)
+
+If the product is Out of stock without the option "Allow order" and cannot be added to the cart: 
+-	a message is displayed informing the user that the product is no longer available: "Product out of stock".
+-	the "Add to basket" CTA is disable and the product cannot be added to cart
+
+![wishlist screen 12](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-12.png)
+
+If order of out of stock products is denied and if the selected combination is out of stock and if there are other available combination (in stock or out of stock with allowed order) for the same product: 
+-	a message tells the user that the product is available in other options "Product available with different options"
+-	the product image and the Add to cart CTA are disabled
+
+![wishlist screen 13](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-13.png)
+
+For a customized product :
+* If the customization is required and if the customization isn't defined (it is still not defined since it is impossible to save the customisation as a wishlist), the user will not be able to add this product to the cart directly from the favorites list. The "Add to cart" CTA is replaced by "Customize". When clicking on the CTA, the user is redirected to the product sheet in order to customise the product, and if it is already filled in, they can add this product as a favorite. 
```

Comment:
```suggestion
* If the customization is required and as the customization is always not defined (since it is impossible to save the customization as a wishlist), the user will not be able to add this product to the cart directly from the favorites list. The "Add to cart" CTA is replaced by "Customize". When clicking on the CTA, the user is redirected to the product sheet in order to customize the product, and if it is already filled in, they can add this product to the cart. 
```